CONTRIBUTING.md

```diff
@@ -15,9 +15,9 @@ Otherwise, go read Docker's
 
 ### Development Environment Setup
 
-Swarm is written in [the Go programming language](http://golang.org) and manages its dependencies using [Godep](http://github.com/tools/godep).  This guide will walk you through installing Go, forking the Swarm repo, building Swarm from source and contributing pull requests (PRs) back to the Swarm project.
+Swarm is written in [the Go programming language](http://golang.org) and manages its dependencies using [govendor](https://github.com/kardianos/govendor).  This guide will walk you through installing Go, forking the Swarm repo, building Swarm from source and contributing pull requests (PRs) back to the Swarm project.
```

Comment:
It'd be better to provide version for the switch between Godep and Govendor. Users running older versions should check CONTRIBUTING.md in the source.